frame/example/src/weights.rs

```diff
@@ -0,0 +1,100 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Autogenerated weights for pallet_example
```

Comment:
in order to generate this file, I guess you add the palelt to the node-runtime, would be good to also explain to user how it was generated. I'm not sure where either here or in the module import in lib.rs
Maybe in both.

Reply:
I have added comment at the top of `benchmarking.rs` file referring to devhub doc on running the benchmark and generating this weight implementation.